src/realm/spec.cpp

```diff
@@ -146,7 +146,7 @@ MemRef Spec::create_empty_spec(Allocator& alloc)
         spec_set.add(v); // Throws
         dg_2.release();
     }
-    spec_set.add(0); // Subspecs array
+    spec_set.add(0); // Nested collections array
```

Comment:
I was expecting some extra field to be added to the file, but it seems we had already considered this in the past? Is this correct?

Reply:
This was a position used in Core-5.